### DISCUSSION TITLE
Plugins wont open anymore on Windows Server

### DISCUSSION THREAD

**Discussion by TheRiddler303** [apohl79/audiogridder#464] [Help]
Hello,
i was running Audiogridder v1.1.1 without any problems till yesterday.
Yesterday it stopped opening the GUI of the Plugins on the Server. Any Plugin shortly pops up for a second and closes down straight.
So it also wont transfer it to my DAW with the AG Pluugin inserted. It happens to all plugins. no matter if its a instrument or an fx plugin.
Also i noticed, when i load Ag Plugin as an insert, it wont play the audio file on that track anymore. it gives a audio buffer error within a few seconds of silence and freeze.
The Plugins will open on the Server Machine, when i diasble the screen capture mode.

I reinstalled the server and the plugin already. reloaded the plugin cache and tried all screen cap options. 
no chance.

My Setup:
Windows Server is running through Parallels on a Macbook Pro, 2,3GhZ i7, 16GB Ram. 
Plugin is on a iMac 3,4 GhZ, i7, 16 GB RAM with Logic Pro X 10.4.8 on OSX 10.13.

Any idea how to fix this, without setup my parallels windows again?

Thanks

**Comment by CaBuffalo** [comment]
I feel it's a problem from your network. Did you install any update or APP
before the problem happened?

Most likely there is something going on between the connection of your DAW
and Win Server.

On Sat, Apr 3, 2021 at 2:37 AM TheRiddler303 ***@***.***>
wrote:

> Hello,
> i was running Audiogridder v1.1.1 without any problems till yesterday.
> Yesterday it stopped opening the GUI of the Plugins on the Server. Any
> Plugin shortly pops up for a second and closes down straight.
> So it also wont transfer it to my DAW with the AG Pluugin inserted. It
> happens to all plugins. no matter if its a instrument or an fx plugin.
> Also i noticed, when i load Ag Plugin as an insert, it wont play the audio
> file on that track anymore. it gives a audio buffer error within a few
> seconds of silence and freeze.
> The Plugins will open on the Server Machine, when i diasble the screen
> capture mode.
>
> I reinstalled the server and the plugin already. reloaded the plugin cache
> and tried all screen cap options.
> no chance.
>
> My Setup:
> Windows Server is running through Parallels on a Macbook Pro, 2,3GhZ i7,
> 16GB Ram.
> Plugin is on a iMac 3,4 GhZ, i7, 16 GB RAM with Logic Pro X 10.4.8 on OSX
> 10.13.
>
> Any idea how to fix this, without setup my parallels windows again?
>
> Thanks
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/464>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6E2FCG4EH3G2WDWNATTG3OVXANCNFSM42KB2C7A>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by TheRiddler303** [comment]
yes i did some settings on little snitch. but i changed those settings in the morning and it was running smooth after that for another 2-3 hours...?! 
network is still connecting, otherwise i could not load the plugin in logic. it just dont open up or seems to send audio through the plugin.

**Comment by apohl79** [comment]
Can you follow https://audiogridder.com/bug-reports/ ?

As it happens with any plugin, please setup a minimal session with 1 plugin, try to open it and close everything.

**Comment by TheRiddler303** [comment]
Thanks, for looking into it.

This is what i did now:
I cleaned my vstplugin folder to 1 plugin and build new database cache on AG Server. i also notice, that loading AG as an instrument, to go through the file menu to load the plugin it always has little freezes and take a good second, before the next menu bounces up. (like vst->synth->xfer->serum) also search function is going really slow. this does not happen on an audio insert. this goes straight and fast click by click through the menus. Not sure if that info is useful. but my error, happens on both instruments as on audio fx plugins. 

today i was only able to connect server on parallels windows to my logic DAW which is on the same macbook. dont have my usually used imac here at home. but this option worked yesterday morning and before without any problems too. so error happens with internal network on the same machine, as through lan connection to a different plugin machine.

i started server on windows, opened up logic on mac, loaded the instrument plugin. GUI popping up on the server for half a second and dissapears after it. unloaded the plugin from logic. quit server on the windwos machine.

here are the log files. havent found a core file on my mac.

cheers

[AG1.1.1 Logs The Riddler303.zip](https://github.com/apohl79/audiogridder/files/6254194/AG1.1.1.Logs.The.Riddler303.zip)
